docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       - RABBITMQ_SECURE_PASSWORD=yes
     volumes:
       - 'rabbitmq_data:/bitnami/rabbitmq/mnesia'
-  db:
+  mysql:
```

Comment:
Knitpicking on naming consistency 😄 .
Since we changed`db` to `mysql` we could probably change the **volume** name from `my-db` to `mysql-db`

Reply:
Nice catch, fixed